### ISSUE TITLE
Export CSV caused Mindforger to stop responding

### ISSUE THREAD

**Issue by Iznik285** [dvorka/mindforger#1291]
**Steps to recreate:**

Click on Mind / Export / CSV the dialog box comes up.
Click on Export. The file is exported but Mindforger then freezes.
The only way to continue is to close the application and restart.

I am running Windows 10 / MF 1.52.0

I think this was first reported in Issue #1001 but still exists.

**Comment by dvorka** [comment]
@Iznik285 Yes, it's known bug which I didn't fix yet, however, I will give it much bigger priority. Thank you for raising this problem!
